packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -221,14 +260,23 @@ function HierarchicalTermSelector({ noParentId = NO_PARENT_VALUE, taxonomy }) {
         onChange={handleClickCategory}
         noOptionsText={taxonomy.labels.not_found}
       />
+      <LinkButton
+        ref={toggleRef}
+        aria-expanded={false}
+        onClick={handleToggleNewCategory}
+        _showAddNewCategory={showAddNewCategory}
+      >
+        {taxonomy.labels.add_new_item}
+      </LinkButton>
```

Comment:
Since we wanted to move focus back to this, I had to leave it mounted when `showAddNewCategory` changes otherwise the ref becomes null. This meant I had to hide with css. I am sure there are other ways, but I figured a button wasn't too big a deal to leave mounted, thoughts?

Reply:
this makes sense to me, totally fine.